stout_a05a.js

```diff
@@ -0,0 +1,29 @@
+/*Kyle Stout
+  stout_a05a
+  INFO 2124WW
+  Thoendel
+  1-19-2020
+*/
+

```

Comment:
No line-by-line comments means an automatic 10% deduction as per the assignment.
-2